src/types.rs

```diff
@@ -48,7 +48,7 @@ use std::marker::PhantomData;
 use std::num::NonZeroU32;
 use std::os::unix::io::RawFd;
 
-pub use sys::__kernel_rwf_t as RwFlags;
+pub type RwFlags = u32;
```

Comment:
Since upstream already uses u32 directly, I think this can be marked as deprecated.